src/api/test/functional/webui/spider_test.rb

```diff
@@ -151,7 +151,7 @@ def test_spider_anonymously
     crawl
     ActiveRecord::Base.clear_active_connections!
 
-    @pages_visited.keys.length.must_be :>, 700
+    @pages_visited.keys.length.must_be :>, 500
```

Comment:
Seems that changed because we [removed the noscript element](https://github.com/bgeuken/open-build-service/commit/7b9d3b8b879cce1d068fb37c3c6ffb7a536221c5#diff-b9c41081d49ec85416f6d65bf80ee796L51)